frontend/src/app/pages/summary/balance/balance.component.scss

```diff
@@ -3,10 +3,10 @@
 }
 
 .label {
-  font-size: small;
+  font-size: 1.25em;
```

Comment:
What does `em` stand for in this case? Is it normalized anywhere? If not, maybe it should be?

Reply:
Nice